README.md

```diff
@@ -97,50 +74,33 @@ For more information on Jenkins and Docker setup read the corresponding document
 # Building & Running the application locally
 
 First, start the zookeeper, and kafka services that is described in the ```doc/runKafka.md``` file.
-
-Then, to build the application, issue the following command in the parent project folder
-
+Then set up the postgres database as described in ```doc/postgres.md``` file.
```

Comment:
Could you please link the doc here? The navigation will be easier

Reply:
Yes, good point. Edited.